sdk/src/main/java/com/horizen/utils/Ed25519.java

```diff
@@ -0,0 +1,43 @@
+package com.horizen.utils;
+
+import org.bouncycastle.crypto.params.Ed25519PrivateKeyParameters;
+import org.bouncycastle.crypto.params.Ed25519PublicKeyParameters;
+import scorex.crypto.hash.Sha256;
+
+public class Ed25519 {
```

Comment:
class should be final.
`Ed25519` is a class with only static methods. Make constructor private?

Reply:
Done